### PR TITLE
Fix/117627 new ticket issue

### PR DESCRIPTION
**Pull request by paulmskim**
For ticket: https://central.tri.be/issues/117627

### CODE REVIEW

**Comment by tr1b0t** [comment]
Closing this Pull Request because it is pull requested against the wrong branch. The Central ticket is for *Sprint 6*, but this was pull requested against *release/M18.18*.

File: src/modules/data/blocks/ticket/selectors.js [hunk 1]
```diff
@@ -113,21 +113,18 @@ export const getTicketsCount = createSelector(
...
-);
-
 export const hasTickets = createSelector(
 	[ getTicketsCount ],
 	( count ) => count > 0,
 );
 
+export const hasCreatedTickets = createSelector(
+	[ getTicketsArray ],
+	( tickets ) => tickets.reduce( ( hasCreated, ticket ) => (
```
Comment (nealfennimore) [inline src/modules/data/blocks/ticket/selectors.js]: Are we trying to see if every item has been created, or just some of them? We can probably replace it with the `.every` or `.some` method if it's either of those cases

Comment (mitogh) [reply src/modules/data/blocks/ticket/selectors.js]: We are trying to look if at least one is  `true`

Comment (paulmskim) [reply src/modules/data/blocks/ticket/selectors.js]: ⬆️

**Pull request merged**